bessctl/commands.py

```diff
@@ -1899,6 +1930,16 @@ def tcpdump_module(cli, module_name, direction, gate, opts):
                     gate, opts, 'tcpdump', cli.bess.tcpdump)
 
 
+@cmd('tcpdump_request MODULE FIFO [DIRECTION] [GATE] [TCPDUMP_OPTS...]',
+     'Request packet capture on a gate')
+def tcpdump_request_module(cli, module_name, fifo, direction, gate, opts):
+    # TODO(nlsun) It may be better to implement this as a flag in the `tcpdump`
+    # command, but there doesn't seem to be a mechanism for adding flags at
+    # this time.
+    _capture_request_module(cli, module_name, direction,
+                            gate, opts, 'tcpdump', cli.bess.tcpdump, fifo)
+
```

Comment:
I agree with your TODO comments. In my opinion, clearing up those commands is quite important as it is open to users.  Is it possible that 'tcpdump_request'  integrated with 'tcpdump' along with integration of 'capture_request_module' and 'capture_module'? And follows the existing command syntax? For example, existing cmd does not use '_' but multiple tokens.